package.json

```diff
@@ -9,21 +9,44 @@
     "react-scripts": "0.9.5"
   },
   "dependencies": {
+    "better-console": "^1.0.1",
     "csvexport": "^1.0.2",
     "date-fns": "^1.21.0",
     "director": "^1.2.8",
     "gocommerce-js": "^3.9.0",
     "gulp": "^3.9.1",
+    "gulp-autoprefixer": "^4.0.0",
```

Comment:
I removed semantic ui as a dependency as it was causing issues and shouldn't have been added here. So I needed to add the libraries that the semantic ui build tasks depended on directly.